bin/cli.js

```diff
@@ -25,15 +25,16 @@ var argv = require("yargs")
     "elm-test-args",
     'Pass arguments to elm-test. f.e. `--elm-test-args="--report=junit"`'
   )
-  .coerce("elm-test-args", function(arg) {
+  .coerce("elm-test-args", function (arg) {
     if (typeof arg === "string") return arg.split(" ");
     return [];
   })
   .default("elm-test-args", []).argv;
 
 var model = runner.init(argv);
 
-runner.run(model, function(warnings) {
+runner.run(model, function (warnings) {
+  console.log("warnings", warnings);
```

Comment:
```suggestion
  console.warn("warnings", warnings);
```

AFAIK it goes to stderr in node, which is probably more desirable.